apps/consultation-portal/screens/Home/utils/getDateForComparison.ts

```diff
@@ -1,8 +1,10 @@
-export const getDateForComparison = (date: Date | string) => {
+const getDateForComparison = (date: Date | string) => {
   try {
     return new Date(date).getTime()
   } catch (e) {
     console.error(e)
     return null
   }
```

Comment:
væri ekki sniðugt að hafa öll date functions í einu skjali?

Reply:
getDateForComparison moved to dateFormatter which in turn was renamed to dateFunctions